examples/lib/defaultHtmlStepUi.js

```diff
@@ -295,7 +295,8 @@ function DefaultHtmlStepUi(_sequencer, options) {
     $step('img').show();
     $stepAll('.load-spin').hide();
     $step('.load').hide();
-
+    $stepAll('.download-btn').off('click');
```

Comment:
Why is this done? And why is it `$stepAll` and not `$step`?

Reply:
The event listener was getting called multiple times. This could be because we are adding event listeners everytime we change the value from default. So I thought maybe to clear the download button from all event listeners before a listener is added when value is changed from default.

Reply:
The code is working for `$step` also but as event listeners were added using `$stepAll` I thought one might have to remove them using `$stepAll`. I may be wrong here. I will study more about scope query and update.